.rubocop.yml

```diff
@@ -100,6 +101,164 @@ Packaging/RequireHardcodingLib:
 Packaging/RequireRelativeHardcodingLib:
   Enabled: true
 
+Performance:
+  Enabled: true
```

Comment:
should I remove this? This will implicitly enable all cops under `Performance`

Reply:
No. It won't because we have `DisabledByDefault: true` set. This means that cops are now opt-in vs opt-out. On future Rubocop and sibling gem releases, new cops will be disabled and a log message will appear when running rubocop. Easier to see this in action. Here is sample output (partial) from running the rubocop command in CI from another repo with a rubocop-rails Dependabot PR.

```
...
  The following cops were added to RuboCop, but are not configured. Please set Enabled to either `true` or `false` in your `.rubocop.yml` file.
  
  Please also note that you can opt-in to new cops by default by adding this to your config:
    AllCops:
      NewCops: enable
  Rails/DangerousColumnNames: # new in 2.21
    Enabled: true
  Rails/RedundantActiveRecordAllMethod: # new in 2.21
    Enabled: true
  Rails/UnusedRenderContent: # new in 2.21
    Enabled: true
  For more information: https://docs.rubocop.org/rubocop/versioning.html
...
```

Reply:
Ok, so I will do the same in capybara

Reply:
Great thanks!